From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 24 Dec 2020 16:02:15 +0100
Subject: Use sphinx_rtd_theme theme as a replacement of furo theme

furo theme is not available right now in Debian. So use a replacement
theme.

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/conf.py | 15 +++++----------
 1 file changed, 5 insertions(+), 10 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 29aa568..b99f183 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 import os
 import sys
+import sphinx_rtd_theme
 
 from streamlink import __version__ as streamlink_version
 
@@ -24,7 +25,8 @@ extensions = [
     'ext_github',
     'ext_plugins',
     'ext_releaseref',
-    'recommonmark'
+    'recommonmark',
+    'sphinx_rtd_theme'
 ]
 
 autosectionlabel_prefix_document = True
@@ -115,7 +117,7 @@ intersphinx_timeout = 60
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'furo'
+html_theme = 'sphinx_rtd_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
@@ -134,7 +136,7 @@ html_theme_options = {}
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "../icon.svg"
+html_logo = "_static/icon.svg"
 
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
@@ -148,9 +150,6 @@ html_static_path = ['_static']
 
 html_css_files = [
     'styles/custom.css',
-    'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/fontawesome.min.css',
-    'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/solid.min.css',
-    'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/brands.min.css',
 ]
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
@@ -160,12 +159,8 @@ html_css_files = [
 # Custom sidebar templates, maps document names to template names.
 html_sidebars = {
     '**': [
-        'sidebar/scroll-start.html',
         'sidebar/brand.html',
-        'sidebar/search.html',
-        'sidebar/navigation.html',
         'sidebar/github-buttons.html',
-        'sidebar/scroll-end.html',
     ]
 }
 
